src/custom/pages/Claim/index.tsx

```diff
@@ -175,7 +175,7 @@ export default function Claim() {
   // on account change
   useEffect(() => {
     if (!isSearchUsed && account) {
-      setActiveClaimAccount(account)
+      setActiveClaimAccount('0xD5e900280Eb1aDe4E583c2bF2414be247E298435')
```

Comment:
why hardcoding this?

Reply:
Fixed, just for development so I don't have to insert one while working on investment flow